tutorials/Numerics/TimeStepping/explicit_lsrk.jl

```diff
@@ -0,0 +1,190 @@
+# \end{align}
+# ```
+#
+# where ``\boldsymbol{\mathcal{T}}(\boldsymbol{Q}^i)`` is the evaluation of the
+# right-hand side tendency at the stage value ``\boldsymbol{Q}^i``, defined at
+# each stage of the RK method:
+#
+# ```math
+# \begin{align}
+# 	\boldsymbol{Q}^i = \boldsymbol{q}^{n} +
+#     \Delta t \sum_{j=1}^{s} a_{i,j}
+#     \mathcal{T}(\boldsymbol{Q^j}).
+# \end{align}
+# ```
+#
+# The first stage is initialized using the field at the previous time step:
+# ``\boldsymbol{Q}^{1} = \boldsymbol{q}^n``.
+#
+# In the above expressions, we define
+# ``\boldsymbol{A} = \lbrace a_{i,j} \rbrace \in \mathbb{R}^{s\times s}``,
+# ``\boldsymbol{b} = \lbrace b_i \rbrace \in \mathbb{R}^s``, and
+# ``\boldsymbol{c} = \lbrace c_i \rbrace \in \mathbb{R}^s`` as the
+# characteristic coefficients of a given RK method. This means we can
+# associate any RK method with its so-called *Butcher tableau*:
+#
+# ```math
+# \begin{align}
+#     \begin{array}{c|c}
+#         \boldsymbol{c} &\boldsymbol{A}\\
+#         \hline
+#         & \boldsymbol{b}^T
+#         \end{array} =
+#         \begin{array}{c|c c c c}
+#         c_1 & a_{1,1} & a_{1,2} & \cdots & a_{1,s}\\
+#         c_2 & a_{2,1} & a_{2,2} & \cdots & a_{2,s}\\
+#         \vdots & \vdots & \vdots & \ddots & \vdots\\
+#         c_s & a_{s,1} & a_{s,2} & \cdots & a_{s,s}\\
+#         \hline
+#         & b_1 & b_2 & \cdots & b_s
+#     \end{array}.
+# \end{align}
+# ```
+#
+# The vector ``\boldsymbol{c}`` is often called the *consistency vector*,
+# and is typically subjected to the row-sum condition:
+#
+# ``
+# c_i = \sum_{j=1}^{s} a_{i,j}, \quad \forall i = 1, \cdots, s.
+# ``
+#
+# This simplifies the order conditions for higher-order RK methods.
+# For more information on general RK methods, we refer the interested reader
+# to Ch. 5.2 of [Atkinson2011](@cite).
+#
+# ### [Low-storage Runge-Kutta methods](@id lsrk)
+# `ClimateMachine.jl` contains the following low-storage methods:
+#   - Forward Euler (`LSRKEulerMethod`)
+#   - A 5-stage 4th-order Runge-Kutta method of Carpenter and Kennedy (`LSRK54CarpenterKennedy`)
+#   - A 14-stage 4th-order Runge-Kutta method developed by Niegemann, Diehl, and Busch (`LSRK144NiegemannDiehlBusch`).
+#
+# To start, let's try using the 5-stage method: `LSRK54CarpenterKennedy`.
+
+# As is the case for all explicit methods, we are limited by the fastest
+# propogating waves described by our governing equations. In our case,
+# these are the acoustic waves (with approximate wave speed given by the
+# speed of sound of 343 m/s).
+# For the rising bubble example used here, we use 4th order polynomials in
+# a discontinuous Galerkin approximation, with a domain resolution of
+# 125 meters in each spatial direction. This gives an effective
+# minimanl node-distance (distance between LGL nodes) of 86 meters
+# over the entire mesh. Using the equation for the explcit time-step above,
+# we can determine the ``\Delta t`` by specifying the desired Courant number ``C``.
```

Comment:
Could we change C to CFL? It is more self-explaining and is used in most of the current experiment drivers

Reply:
Thanks for pointing that out. I thought it some other parts of the code Courant was used (see [here](https://clima.github.io/ClimateMachine.jl/previews/PR1938/APIs/Numerics/DGMethods/Courant/)) and that's how I heard most people referring to it here. I am going to rename it